.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "deps/libtuv"]
 	path = deps/libtuv
 	url = https://github.com/DLTcollab/libtuv.git
+[submodule "deps/rabbitmq-c"]
+	path = deps/rabbitmq-c
+	url = https://github.com/DLTcollab/rabbitmq-c.git
```

Comment:
Since there is no visible change against `rabbitmq-c`, why did you fork from RabbitMQ C client?

Reply:
I only want to reserve a rabbitmq-c version for dcurl use. If we do not need it, I modify it.